app/controllers/APIController.scala

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import java.time.LocalDateTime
+import javax.inject._
+import play.api._
+import play.api.mvc._
+import play.api.libs.json._
+import play.api.Environment._
+
+import java.time.format.DateTimeFormatter
+
+/**
+ * This controller creates an `Action` to handle HTTP requests to the
+ * application's home page.
+ */
+@Singleton
+class APIController @Inject()(val controllerComponents: ControllerComponents) extends BaseController {
+
+  /*
+   * method that returns a json object with the following information:
+   * { "service" : "pass-bakery"
+   *   "environment" : string (production or development run, based on play fw env config)
+   *   "serverTime" : string (local time from dateTime library in ISO 8601 format)
+   * */
+  def getStatus() = Action { implicit request: Request[AnyContent] =>
+    
+    play.api.Mode.Prod
+
+    // construct the json
+    val json: JsValue = Json.obj(
+      "service" -> "pass-bakery",
+      "environment" -> play.api.Mode.toString,
+      "test" -> play.api.Mode.Dev.toString,
+      "serverTime" -> LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"))
+    )
```

Comment:
Good start! A thing to keep in mind is that it is always better to define the models you return than build JSON or other serialized formats ad hoc. In this case, I would suggest defining a [case class](https://docs.scala-lang.org/tour/case-classes.html) in file in the `app/models` directory that describes what you will send to the the user/api consumer in this case, and instantiate the case class here with any values that need to be defined or calculated when the response is returned to the user

Reply:
added the case class model under a new models directory and implemented it in the API controller